README.md

```diff
@@ -210,8 +209,6 @@ Example:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
 - [ ] **[new_feature_1]**
 - [ ] **[new_feature_2]**
 - [ ] **[new_feature_3]**
```

Comment:
- [ ] It will be better to add 1-3 sentence about the features you will work on them in the future like drag and drop something like this